cmd/upgrade.go

```diff
@@ -95,11 +100,35 @@ func upgradeFunc(cmd *cobra.Command, args []string) {
 		return
 	}
 
+	// read common ignore file if it exists
+	ignorePatterns := make([]string, 0)
+	if data, err := os.ReadFile(filepath.Join(target, recipe.IgnoreFileName)); err == nil {
+		ignorePatterns = append(ignorePatterns, strings.Split(string(data), "\n")...)
+	} else if !errors.Is(err, fs.ErrNotExist) {
+		// something else happened than trying to read an ignore file that does not exist
+		cmd.PrintErrln("failed to read ignore file", err)
```

Comment:
If the user has not created .jalapenoignore, does it still state it as an error here?